README.md

```diff
@@ -172,9 +176,10 @@ sudo zypper in -y \
     libQt5DBus-devel \
     libQt5Charts5-devel \
     libqt5-qtbase-devel \
-    libopenssl-1_0_0-devel \
-    python3-devel \
-    python-hwdata
+    libopenssl-devel \
+    python311-devel \
+    python311-hwdata \
```

Comment:
Why `python311` and not `python3` ?

Reply:
openSUSE has own packaging rules for Python and building process. We don't have something like python3-devel, we have package python-devel, that provides 3 packages (for python 3.9, 3.10 and 3.11). Current main Python in openSUE Tumbleweed is 3.11 

Link for packaging rules:
https://en.opensuse.org/openSUSE:Packaging_Python#What_is_single-spec?

Reply:
In fact of that, we have 4 different pythons with modules in repo at that point, that marked by version (python39-notify2 for example, not python3)